coding-concepts.js

```diff
@@ -9,28 +9,29 @@
 // --------------------1) What will this log?
 
 const cohort = "Alpha 2024"
-// console.log(cohort.split(" "))
+//console.log(cohort.split(" "))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "Alpha" "2024"
+// b) Verify and explain: [ 'Alpha', '2024' ]. so im right but wrong! had to remember that .split returns a new array of the string. it also puts the string into a substrings but within side a new array. 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+//console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: undefined. because name is the parameter not the variable. and there is no value to a variable with name. so using string interpolation wont work.
+// b) Verify and explain: got this one right. these is just no value here with a name when you are console logging. you would need a variable called name with a value for this to work. 
```

Comment:
Not quite.  Remember that `name` is the parameter.  "Learn Student" is the argument, so we don't need a variable.  The issue is there is no `return`.